### PR TITLE
[2.7] bpo-22392: Improve documentation for __getinitargs__ 

### PR DESCRIPTION
**Pull request by daxlab**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](http://bugs.python.org/) (b.p.o) to verify you have signed the CLA. This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://cpython-devguide.readthedocs.io/pullrequest.html#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: Doc/library/pickle.rst [hunk 1]
```diff
@@ -418,10 +418,11 @@ Pickling and unpickling normal class instances
    When a pickled class instance is unpickled, its :meth:`__init__` method is
    normally *not* invoked.  If it is desirable that the :meth:`__init__` method
    be called on unpickling, an old-style class can define a method
-   :meth:`__getinitargs__`, which should return a *tuple* containing the
+   :meth:`__getinitargs__`, which should return a *tuple* of positional
    arguments to be passed to the class constructor (:meth:`__init__` for
-   example).  The :meth:`__getinitargs__` method is called at pickle time; the
-   tuple it returns is incorporated in the pickle for the instance.
+   example). However, it doesn't support ``kwargs``. The :meth:`__getinitargs__`
```
Comment (serhiy-storchaka) [inline Doc/library/pickle.rst]: ``kwargs`` is a jargon. I think it would be better to say that keyword arguments are not supported.

**Comment by daxlab** [comment]
@serhiy-storchaka changed.

**Review by serhiy-storchaka — APPROVED** [review]

**Comment by serhiy-storchaka** [comment]
Signing the CLA isn't required for such simple change, but would be nice if you sign it for your future contributions.

**Comment by daxlab** [comment]
CLA signed. @serhiy-storchaka can we merge this ?

**Comment by Mariatta** [comment]
Thanks @daxlab for your first contribution to CPython! :tada:

**Pull request merged**